en/docs/consuming-services/create-a-subscription.md

```diff
@@ -1,20 +1,32 @@
+# Subscribe to an API
 
-You must subscribe to a published API to use it in your application. New API subscriptions cover all minor versions within the subscribed API’s major version.
+To use a published API in your application, you must subscribe to it. When you subscribe to an API, your subscription covers all minor versions within the API's major version.
 
-The subscription process is designed to ensure the secure authentication of API requests via application keys. Alternatively, you can generate credentials for an API without an explicit subscription to an application. However, this approach limits the capability to control advanced configurations such as access token expiry time, revoke token expiry time, ID token expiry time, and enabling access to the API without a secret. Generating keys in the API is recommended for testing or short-term usage but not for long-term production usage. 
+The subscription process ensures secure authentication of API requests using application keys. While you can generate credentials for an API without subscribing to an application, this approach limits advanced configuration options such as access token expiry time, revoke token expiry time, ID token expiry time, and enabling access to the API without a secret. Generating keys directly in the API is suitable for testing or short-term use but is not recommended for long-term production usage.
 
-To subscribe to an API via an application, follow the steps given below: 
+To subscribe to an API via an application, follow these steps:
 
-1. Sign in to the [Choreo Developer Portal](https://devportal.choreo.dev).
-2. In the Developer Portal header, click **Applications**.
-3. On the **My Applications** page, click on the application with which you want to subscribe to an API.
-4. In the left navigation menu, click **Subscriptions**. 
-5. In the **Subscription Management** pane that opens, click **+ Add APIs**. 
-6. Click **Add** to subscribe to an API. Depending on your requirement, you can subscribe to one or more APIs.
+1. **Sign in to the Choreo Developer Portal**  
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. See https://developers.google.com/style/text-formatting and https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#text-formatting-guidelines.
Any specific reason to do this?